bitsail-connectors/connector-kafka/src/main/java/com/bytedance/bitsail/connector/kafka/constants/KafkaConstants.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022-2023 Bytedance Ltd. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.connector.kafka.constants;
+
+public class KafkaConstants {
+  public static final String CONNECTOR_TYPE_VALUE_KAFKA = "kafka";
+  public static final String CONNECTOR_TYPE_VALUE_KAFKA010X = "kafka010x";
+  public static final String CONNECTOR_TYPE_VALUE_KAFKA220 = "kafka220";
+  public static final String CONNECTOR_NAME_VALUE_KAFKA_SOURCE = "kafka_source";
+  public static final String CONNECTOR_NAME_VALUE_KAFKA_SINK = "kafka-sink";
+  public static final String CONNECTOR_VERSION_VALUE_08 = "0.8";
+  public static final String CONNECTOR_VERSION_VALUE_09 = "0.9";
+  public static final String CONNECTOR_VERSION_VALUE_010 = "0.10";
+  public static final String CONNECTOR_VERSION_VALUE_011 = "0.11";
+  public static final String CONNECTOR_VERSION_VALUE_UNIVERSAL = "universal";
+  public static final String CONNECTOR_TOPIC = "connector.topic";
```

Comment:
Hi, @garyli1019 , i have a question, how will we support different versions of kafka here. BTW, I noticed that new flink-kafka-connector only uses the latest version of the [kafka client](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/connectors/datastream/kafka/) and ensure that modern Kafka clients are backwards compatible with broker versions 0.10.0 or later.

Reply:
good question, maybe we should do this as well.